portality/formcontext/emails.py

```diff
@@ -200,25 +213,36 @@ def send_publisher_inprogress_email(application):
     """Tell the publisher the application is underway"""
     journal_title = application.bibjson().title
 
-    owner = models.Account.pull(application.owner)
-    if owner is None:
-        raise app_email.EmailException("Application {x} does not have an owner, cannot send email".format(x=application.id))
+    contact_name = application.get_latest_contact_name()
+    contact_email = application.get_latest_contact_email()
+    send_list = [(contact_name, contact_email,
+                  Messages.SENT_JOURNAL_CONTACT_IN_PROGRESS_EMAIL,
+                  Messages.NOT_SENT_JOURNAL_CONTACT_IN_PROGRESS_EMAIL)]
 
-    # This is to the publisher contact on the application
-    publisher_name = owner.name
-    publisher_email = owner.email
+    owner = models.Account.pull(application.owner)
+    if owner is not None:
+        send_list.append((owner.name, owner.email,
```

Comment:
I find this a bit of a weird data structure to use for this - you have a tuple of name, email, and 2 messages - but these are unlabelled and undocumented.

Reply:
Fair enough, switching to using a dict for tracking the values.